README.md

```diff
@@ -21,37 +21,32 @@ lane :beta do
   match
   testflight
   sh "./customScript.sh"
-
   slack
 end
 ```
 
-To deploy a new beta version of your app
-```
-fastlane beta
+Then to deploy a new 'beta' version of your app just run
 ```
-
-To get started with `fastlane`, follow the [guide](https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Guide.md).
-
+fastlane beta```. :rocket:
 
               |  fastlane
 --------------------------|------------------------------------------------------------
-:sparkles: | Connect all iOS and Android build tools into one workflow (both `fastlane` tools and third party tools)
-:monorail: | Define different `deployment lanes` for App Store deployment, beta builds or testing
-:ship: | Deploy from any computer, including a CI-server
+:sparkles: | Connect iOS, Mac, and Android build tools into one workflow (both `fastlane` tools and third party tools)
+:monorail: | Define different `deployment lanes` for app store deployment, beta builds, or testing
```

Comment:
Oh right, I remember this change now… is this line meant to be iOS specific? (that's what uppercase is for, compared to the general distribution concept)